src/main/java/seedu/duke/parsermanager/pairunpair/PairUnpairParser.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.parsermanager.pairunpair;
+
+import seedu.duke.exception.pairunpair.PairUnpairNotIntegerException;
+import seedu.duke.parsermanager.Parser;
+
+import java.util.ArrayList;
+
+public abstract class PairUnpairParser extends Parser {
+
+
+    protected boolean isFlagPresent(int flagIndexPosition) {
+        return (flagIndexPosition != FLAG_ABSENT_RETURN_VALUE);
+    }
+
+
+    protected ArrayList<Integer> convertPairUnpairCommandDetailsToInteger(ArrayList<String> pairDetailsString)
```

Comment:
You can consider shortening the method name.
Since this function is in the scope of PairUnpairParser already, maybe the "PairUnpair" can be removed?